ui.apps/src/main/content/jcr_root/apps/core/fd/components/form/textinput/v1/textinput/textinput.html

```diff
@@ -32,7 +32,7 @@
      data-cmp-adaptiveformcontainer-path="${formstructparser.formContainerPath}">
     <div data-sly-call="${label.label @componentId=component.id, labelValue=text.label.value, labelVisible=text.label.visible, bemBlock='cmp-adaptiveform-textinput'}" data-sly-unwrap></div>
     <textarea data-sly-test.useTextarea="${text.multiLine}"
-              title="${text.tooltipVisible ? '' : renderer.tooltipContent}"
```

Comment:
there should be a way to configure `tooltipVisible` in edit dialog. cc @vdua  @rismehta

Reply:
Its already there. In the 'Help Content' tab, there is a checkbox 'Always show short description'.

Reply:
`tooltipVisible` variable need not be checked for displaying the hover text in the component.
This variable defines whether to show the text at the bottom of the component. This is how it used to behave in AF1 as well.

Reply:
yes tooltipVisible shows the tooltip content at the bottom of the component but the tooltip will not be shown. If tooltipVisible is false only then tooltip is shown, this is how it works in af1 as far as i know and the same thing we have tried to replicate in af2. Please correct if I am mistaken here. @nit23uec @vdua

Reply:
The flag is to always show short description. If not always shown, then it will work as a tooltip otherwise it will be visible at the bottom of the field